### ISSUE TITLE
too high CPU load

### ISSUE THREAD

**Issue by hq9000** [the-synister/the-source#14]
Hi there. Love the sound of the synth. There's a problem though. It eats way too much CPU. 

E.g. with piano sth. preset, just pressing one key like 4 times a second for a couple of seconds gets my "RT CPU" graph in Reaper reach 100% and the audio begins to click.

Other synths with the same poliphony (synth1 tried) barely bring any load.

please help )

**Comment by hq9000** [comment]
win7 32bit reaper 5.23

**Comment by mahal-tu** [comment]
you are totally right - the cpu load is extraordinary high. This is because the synth has not been optimized at all until now. We will address this in a future release
